apollo/src/main/graphql/com/hedvig/android/owldroid/graphql/providerStatus.graphql

```diff
@@ -0,0 +1,8 @@
+query providerStatus {
```

Comment:
Do we need a separate query for this? Is it because we cant combine it with a subscription?

Reply:
The subscription does not actually give us the human-readable insurance name :( Has to do with the fact that insurely itself has a separate endpoint to provide this information. I will give you more context in Slack.